src/hatti/views/map.cljs

```diff
@@ -329,6 +458,49 @@
                    :on-click (click-fn #(om/update! geofield field))}]
                  (get-label field) [:br]])]]]))))))
 
+(defn map-layer-selector
+  [cursor owner]
+  (reify
+    om/IInitState
+    (init-state [_]
+      {:expanded false
+       :styles ["basic" "streets" "bright" "light" "dark" "satellite"]
```

Comment:
yea styles and base-maps refer to the same thing here. The old base maps will not work with MapboxGL.